themes/dulcie/dulcie.theme.bash

```diff
@@ -77,7 +77,7 @@ dulcie_prompt() {
   printf "\033]0;%s@%s:%s\007" "${USER}" "${HOSTNAME%%.*}" "${PWD/#$HOME/\~}"
 
   # Open the new terminal in the same directory
-  declare -f __vte_osc7 > /dev/null && __vte_osc7
+  _command_exists __vte_osc7 && __vte_osc7
```

Comment:
Since this was specifically testing for being a function, you might want to use [`_is_function`](https://github.com/Bash-it/bash-it/blob/master/lib/helpers.bash#L167)